conan/tools/meson/toolchain.py

```diff
@@ -380,11 +391,11 @@ def _context(self):
             # https://mesonbuild.com/Machine-files.html#project-specific-options
             "project_options": {k: to_meson_value(v) for k, v in self.project_options.items()},
             # https://mesonbuild.com/Builtin-options.html#directories
-            # TODO : we don't manage paths like libdir here (yet?)
             # https://mesonbuild.com/Machine-files.html#binaries
             # https://mesonbuild.com/Reference-tables.html#compiler-and-linker-selection-variables
-            "c": self.c,
-            "cpp": self.cpp,
+            "c": _sanitize_format(self.c),
+            "cpp": _sanitize_format(self.cpp),
+            "ld": _sanitize_format(self.ld),
```

Comment:
@franramirez688 According to the Meson documentation [here](https://mesonbuild.com/howtox.html#set-linker), I didn't think using a multivalue argument for the linker options made sense sense it would have to either `gold` or `lld` or `bfd` for Clang / GCC and only for MSVC / Clang-Cl should it be an executable. I think this is pretty different behavior than how autotools make use of the `LD` environment variable.